server/pbenchinacan/container-build.sh

```diff
@@ -75,10 +70,14 @@ buildah run $container rm /etc/logrotate.d/nginx
 # Setup the Pbench Server systemd service.
 buildah run $container cp ${SERVER_LIB}/systemd/pbench-server.service \
     /etc/systemd/system/pbench-server.service
+buildah run $container cp ${SERVER_LIB}/pbench/system/pbench-index.service \
+    ${SERVER_LIB}/pbench/system/pbench-index.timer \
+    /etc/systemd/system/
```

Comment:
Perhaps do this with a single command?
```sh
# Setup the Pbench Server systemd service.
buildah run $container cp \
    ${SERVER_LIB}/systemd/pbench-server.service \
    ${SERVER_LIB}/pbench/system/pbench-index.service \
    ${SERVER_LIB}/pbench/system/pbench-index.timer \
    /etc/systemd/system/
```

Reply:
Again, they started out in different locations, and I never thought to merge the streams. Sigh.